rust/connlib/tunnel/src/client.rs

```diff
@@ -385,35 +393,29 @@ impl ClientState {
 
     fn request_access(
```

Comment:
@conectado This function has been unified to be the only place where we now emit `RequestAccess` so that we have a nice log with it every time we send it.